exercises/two-fer/.meta/src/reference/kotlin/Twofer.kt

```diff
@@ -0,0 +1,3 @@
+fun twofer(name: String = "you"): String {
+    return "One for ${if (name.isBlank()) "you" else name}, one for me."
```

Comment:
I think this can be altered to

```kotlin
return "One for $name, one for me."
```

since supplying a blank string as input implies different intent to providing _no_ input.